app/(datasets)/stories/[slug]/page.tsx

```diff
@@ -62,37 +21,13 @@ export default function Blog({ params }: { params: any }) {
 
   return (
     <section>
-      <script
-        type='application/ld+json'
```

Comment:
Having ld json is probably the best practice for seo etc.: https://json-ld.org/ 

I see that the current configuration doesn't reflect the state of the story very well, it might be worth getting it right.